sixtyfps_runtime/corelib/graphics/image.rs

```diff
@@ -11,6 +11,175 @@ LICENSE END */
 use crate::slice::Slice;
 use crate::{SharedString, SharedVector};
 
+/// SharedPixelBuffer is a container for storing image data as pixels, backed by
+/// [`SharedVector`]. That means it is internally reference counted and cheap
```

Comment:
But we don't expose the inner vector at any point, right? So that's an implementation detail

Reply:
I think it's good to mention that to make it easier to understand why it's cheap to clone. But OTOH it's not strictly needed - I'll remove it. We can add it in the unlikely event that somebody asks :)